llvm/lib/CodeGen/StackSlotColoring.cpp

```diff
@@ -13,6 +13,7 @@
 #include "llvm/ADT/BitVector.h"
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/ADT/Statistic.h"
+#include "llvm/CodeGen/LiveDebugVariables.h"
```

Comment:
Could probably avoid the include by using the PassID

Reply:
For that, "char &llvm::passID = pass::ID", need to be added in the respective pass (not presently there in LiveDebugVariables.cpp), followed by exposing it via extern in Passes.h. So, I think this is better option,

Reply:
But those should be there

Reply:
They all should (but the PassID will go away in new PM so it's probably not worth fixing all of the missing instances)

Reply:
So, what should be supposed to be my final take on it!